packages/passport/sdk/src/authManager.ts

```diff
@@ -44,7 +45,12 @@ const authorizeEndpoint = '/authorize';
 const getAuthConfiguration = (config: PassportConfiguration): UserManagerSettings => {
   const { authenticationDomain, oidcConfiguration } = config;
 
-  const store = typeof window !== 'undefined' ? window.localStorage : new InMemoryWebStorage();
+  let store;
+  if (config.crossSdkBridgeEnabled) {
+    store = new IndexedDbStorage();
+  } else {
+    store = typeof window !== 'undefined' ? window.localStorage : new InMemoryWebStorage();
+  }
```

Comment:
This should be more readable if we avoid nesting a ternary within an `if`:
```suggestion
  if (config.crossSdkBridgeEnabled) {
    store = new IndexedDbStorage();
  } else if (typeof window !== 'undefined') {
    store = window.localStorage;
  } else {
    store = new InMemoryWebStorage();
  }
```